documentation/src/pages/recipes/data/recipes/full-stack-project-initializer.yaml

```diff
@@ -0,0 +1,244 @@
+         Create __tests__ directories alongside source files.
+       {% elif backend_framework == "python-fastapi" %}
+         Configure Pytest with pytest-asyncio and httpx.
+         Create tests for endpoints, services, and models.
+         Add integration tests for database operations.
+         Set up test fixtures and mocks.
+       {% elif backend_framework == "go" %}
+         Set up Go testing with testify.
+         Create _test.go files for handlers and services.
+         Add integration tests for API endpoints.
+       {% elif backend_framework == "rust-actix" %}
+         Configure cargo test with actix-web test utilities.
+         Create unit and integration tests.
+         Add test modules in src files.
+       {% endif %}
+       Add test scripts with coverage reporting.
+     {% endif %}
+  {% endif %}
+  
+  4. Quality Tools
+     Create .editorconfig for consistent formatting across editors.
+     Create .vscode/settings.json with language-specific formatters.
+     Create .vscode/extensions.json with recommended extensions.
+     Add lint and test scripts to package.json files.
+     Create check-quality.sh script to run all linters and tests.
+  
+  5. Docker Setup
+     {% if include_docker == "true" %}
+     Create Dockerfile for each component (frontend/backend).
+     Create docker-compose.yml with services for:
+     {% if frontend_framework != "none" %}- Frontend{% endif %}
+     {% if backend_framework != "none" %}- Backend{% endif %}
+     {% if database != "none" %}- {{ database }} database{% endif %}
+     Configure volumes, environment variables, and port mappings.
+     Create .dockerignore file.
+     {% endif %}
+  
+  6. CI/CD Setup
+     {% if include_cicd == "true" %}
+     Create .github/workflows/ci.yml with jobs for:
+     - Checkout and setup
+     - Install dependencies
+     - Run linters
+     {% if include_testing == "true" %}
+     - Run unit tests with coverage
+     - Run integration tests
+     {% endif %}
+     - Build artifacts
+     {% if include_docker == "true" %}- Build Docker images{% endif %}
+     {% endif %}
+  
+  7. Documentation
+     Create comprehensive README.md with:
+     - Project overview and tech stack
+     - Prerequisites and installation
+     - Development and testing instructions
+     - Deployment guide
+     Create CONTRIBUTING.md with development guidelines.
+     Create QUICKSTART.md with common commands.
+     {% if project_type != "frontend" %}Create docs/API.md with endpoint documentation.{% endif %}
+     {% if database != "none" %}Create docs/DATABASE.md with schema info.{% endif %}
+  
+  8. Install and Verify
+     Install all dependencies for frontend and backend.
+     Run linters to verify configuration.
+     Run initial tests to ensure everything works.
+     Display summary of created project structure and next steps.
+  
+  Work through these steps systematically. Use shell commands to create files and directories.
```

Comment:
I'd just add the note here for goose that adds the following:

Detect the operating system that I'm on, and use appropriate disk path separators, being careful to avoid 'escaping' of slash or backslash characters as appropriate for my operating system when interpreting or using string values for filenames.